point location data

Stephen-Gates opened the pull request:
**Work in progress.** Advice requested on:

- correct location for this file in directory structure
- if internal links are needed to each option description and if so, how?
- index.md not updated. Should this go in publishing section?
- markdown-link-refs.html not updated - should it be?

Feel free to edit as I'll be on holidays for 2 weeks.

From the [google doc](https://docs.google.com/document/d/194Ow1JfPsocukSKMJ2JuxwZXLWlz_miT74OuG6wwL-A/edit?usp=sharing) originally posted on the [forum](https://discuss.okfn.org/t/tabular-location-data-what-are-my-options/5590).

Stephen-Gates commented on guides/point-location-data.md:
```diff
@@ -0,0 +1,445 @@
+  </tr>
+  <tr>
+    <td>Dalby </td>
+    <td>[151.2660, -27.1944]</td>
```
Given the comma in the data value, how should it be entered so the CSV file is considered valid?

Stephen-Gates commented:
Also, Should I add example CSV files and DataPackages to GitHub for more complete, testable, examples?

danfowler commented on guides/point-location-data.md:
```diff
@@ -0,0 +1,415 @@
+{
+  "fields": [
+    {
+      "name": "Office)",
```
Typo

danfowler commented on guides/point-location-data.md:
```diff
@@ -0,0 +1,415 @@
+{
+  "fields": [
+    {
+      "name": "Office)",
```
Typo

danfowler commented on guides/point-location-data.md:
```diff
@@ -0,0 +1,415 @@
+* Lat, Lon or Lon, Lat - you choose the order
+* No way to force a pair of coordinates and support missing values.
+    * If you add a required constraint to both, you can’t have a missing location.
+    * If you don’t add required constraint, you could have lat without lon or vise versa.
```
vice versa

danfowler commented on guides/point-location-data.md:
```diff
@@ -0,0 +1,415 @@
+
+## Introduction
+
+This guide explores the options available to represent point location data in a CSV file within a data package.
```
My personal preference is to capitalize Data Package on this site.

danfowler commented:
Hi Stephen, I think this is a really great walkthrough of the various options for storing geospatial point data in a tabular format and the pros/cons of each approach.  Given the variety of options I think this is useful research and the guide is a good vehicle for sharing it.  

Thoughts:

1. @ethanwhite and @henrykironde are researching ways to have a consistent method of packaging geospatial (not just point) data.  I also know that @zackbatist has published geospatial point data in a Data Package and might have his own ideas on the best approach: https://github.com/zackbatist/DObsiSS
2. Given that this will be a more/less static document once pushed (it can obviously be changed, but I'm referring to the friction of updating once new information becomes available), maybe it is worth linking to the [existing issues](https://github.com/frictionlessdata/specs/issues/345) for open questions like "Does the geopoint type validation enforce...".

To your questions:

> - correct location for this file in directory structure

We are redoing the site soon, including guides (https://github.com/frictionlessdata/project/issues/332).  I am open to suggestions for where this could go in the new structure.

> - if internal links are needed to each option description and if so, how?

I don't necessarily think so, but I will keep this in mind while updating the guides.

> - index.md not updated. Should this go in publishing section?

See above for thoughts on redoing guides.  Do you think a publishing section is still helpful?

> - markdown-link-refs.html not updated - should it be?

This will change, so focus on the content.

> - Also, Should I add example CSV files and DataPackages to GitHub for more complete, testable, examples?

When I first wrote the guides, I created this repo: https://github.com/frictionlessdata/example-data-packages

We can update the packages therein and you are free to add your examples as well.

Stephen-Gates commented:
@danfowler thanks for your feedback. I will focus on the content and add examples as you suggest. I hope my last commit addressed the points you raised. I did plan to write a similar guide for more complex geometries once people were happy with this guide. Feedback from others is very welcome 😃

rufuspollock commented:
@Stephen-Gates i think this is great.

@danfowler @Stephen-Gates  I'd recommend getting this up asap and then iterating on the live doc to avoid any risk of the PR getting stale or conflicted.

Stephen-Gates commented:
@rufuspollock @danfowler ready to go. Added a link in Index.md - feel free to adjust if it's not in the right place.

danfowler commented:
Great!  Will review and publish!

danfowler commented:
Thanks again, @Stephen-Gates !
